proposals/NNNN-mysql-nio.md

```diff
@@ -0,0 +1,357 @@
+# MySQLNIO: An NIO-based MySQL Driver
+
+* Proposal: [SSWG-NNNN](https://github.com/swift-server/sswg/tree/mysql-nio-proposal/proposals/NNNN-mysql-nio.md)
+* Author(s): [Gwynne Raskind](https://github.com/gwynne)
+* Review Manager: TBD
+* Status: **Implemented**
+* Implementation: [vapor/mysql-nio](https://github.com/vapor/mysql-nio)
+* Forum Threads: Pitch (Pending), Discussion (Pending)
```

Comment:
I think we had a pitch at some point? maybe by @0xTim?

Reply:
Correct I'll suggest an edit